src/PgSqlHandle.php

```diff
@@ -286,7 +288,7 @@ private function createResult($result)
     public function statementExecute(string $name, array $params): Promise
     {
         return call(function () use ($name, $params) {
-            return $this->createResult(yield from $this->send("pg_send_execute", $name, $params));
+            return $this->createResult(yield from $this->send("pg_send_execute", $name, $params), $name);
```

Comment:
`$name` is not the query SQL.

Reply:
I thought name of statement may be more usefull than nothing. I can remove it